Cargo.toml

```diff
@@ -35,7 +35,7 @@ path = "benches/benches.rs"
 [dependencies]
 protobuf = "1.4"
 quick-error = "0.2"
-clippy = {version = "^0", optional = true}
+clippy = {version = "*", optional = true}
```

Comment:
why change to asterisk here?

Reply:
changed according to [clippy doc](https://github.com/rust-lang-nursery/rust-clippy#optional-dependency), hoping to fix the clippy build failure but it seems that it doesn't work.

Reply:
I remember we used the asterisk before but some guys suggested us to use this.

/cc @overvenus

Reply:
Oh, I remember some guys suggested us not using the asterisk if we want to publish the library to the crate.

Reply:
@siddontang Ok, I reverted it.